sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/MetricsAdvisorAdministrationClient.cs

```diff
@@ -1860,25 +1870,6 @@ Page<NotificationHook> NextPageFunc(string nextLink, int? pageSizeHint)
             return PageableHelpers.CreateEnumerable(FirstPageFunc, NextPageFunc);
         }
 
-        private static void ValidateHookToCreate(NotificationHook hook, string paramName)
-        {
-            Argument.AssertNotNull(hook, paramName);
-            Argument.AssertNotNullOrEmpty(hook.Name, $"{paramName}.{nameof(hook.Name)}");
-
-            if (hook is EmailNotificationHook emailHook)
-            {
-                Argument.AssertNotNullOrEmpty(emailHook.EmailsToAlert, $"{paramName}.{nameof(EmailNotificationHook.EmailsToAlert)}");
```

Comment:
I'm removing the empty collections validation as some collections are allowed to be empty by the service (even when the collection itself is required). We could, alternately, check what the service does on each collection and try to match the behavior, but I think it's not worth the trouble and it starts to enter the realm of validating service's parameters.